test/e2e/testing-manifests/scheduling/nvidia-driver-installer.yaml

```diff
@@ -47,17 +47,43 @@ spec:
       - name: root-mount
         hostPath:
           path: /
+      - name: cos-tools
+        hostPath:
+          path: /var/lib/cos-tools
+      - name: nvidia-config
+        hostPath:
+          path: /etc/nvidia
       initContainers:
+      - image: "ubuntu"
```

Comment:
TODO: pin an image instead of depending on a :latest dockerhub image.

Reply:
Updated with specific ubuntu with sha256